hbase-server/src/main/java/org/apache/hadoop/hbase/master/CatalogJanitor.java

```diff
@@ -218,6 +222,20 @@ int scan() throws IOException {
     }
   }
 
+  void repairUnknownServers() {
+    if(!lastReport.unknownServers.isEmpty()) {
+      // submit HBCKServerCrashProcedure to avoid any corner cases that in-memory region states
+      // mismatches with hbase:meta. in fact if HBCKSCP finds any in-memory region states,
+      // HBCKSCP is basically same as SCP.
+      lastReport.unknownServers.stream().forEach(regionInfoServerNamePair -> {
+        LOG.warn("Submitting HBCKSCP for Unknown Region Server {}",
```

Comment:
should this be at INFO level? if it is at WARN what can an operator do to prevent future warnings? if this is something that is going to happen as a matter of course then INFO is more appropriate.

Reply:
+1, this is expected, the log is just for info purposes

Reply:
good point, I will change it.